plugins/scaffolder-backend-module-rails/src/actions/fetch/rails/index.test.ts

```diff
@@ -73,6 +73,9 @@ describe('fetch:rails', () => {
     createTemporaryDirectory: jest.fn().mockResolvedValue(mockTmpDir),
   };
 
+  mockDir.clear();
+  mockDir.addContent({ template: {} });
```

Comment:
It should be possible to remove these and just keep the call in `beforeEach`

Reply:
removed clear and addContent